react-native-oh-library/react-native-harmony-gesture-handler
# Fix: apply transform matrix to touchableViews entries

## Summary
This PR modifies `touchableViews` entries by multiplying `transform` matrix to `frame`. Previously, `touchableViews` dimensions were taken from layout metrics only - when using libraries which update component instances directly, e.g. `react-native-harmony-reanimated`, this property won't be updated when applying some transformations, causing the `touchableViews` entry to be wrong.

## Test Plan
RNGH test:
1. Install `react-native-harmony-reanimated` following [these steps](https://github.com/react-native-oh-library/usage-docs/blob/master/zh-cn/react-native-reanimated.md).
2. Copy-paste [Pan gesture example](https://docs.swmansion.com/react-native-gesture-handler/docs/gestures/pan-gesture) and see if the square is draggable.
3. Copy-paste [Tap gesture example](https://docs.swmansion.com/react-native-gesture-handler/docs/gestures/tap-gesture), modify it to make the square change its scale and rotation after tap, and see if the transform properties are changed only when tapped on the square.

`react-native-bottom-sheet` test (v4 and v5-alpha versions):
1. Install `react-native-bottom-sheet` library.
2. Test examples available in [library documentation](https://ui.gorhom.dev/components/bottom-sheet) and see if sheets are draggable.

## Checklist
- [x] Project compiles and the tester application works.

## Issue links
Resolves: #73

See merge request rnoh/react-native-harmony-gesture-handler!60

(cherry picked from commit fe8d4257bcd4bbcd6476c096d297949469a1eb2c)
